package.json

```diff
@@ -9,7 +9,8 @@
     "start": "node_modules/.bin/babel-node src/index.js",
     "postinstall": "npm run build",
     "build": "node_modules/.bin/babel -d dist src",
-    "test": "node_modules/.bin/babel-tape-runner src/**/*.test.js"
+    "testonce": "node_modules/.bin/babel-tape-runner src/**/*.test.js | ./node_modules/.bin/tap-spec",
+    "test": "watch 'npm run testonce' src/"
```

Comment:
I think it'd be better if we had the tasks "test" to run all the tests and "watch" for development